enhancements/inventory-object.md

```diff
@@ -0,0 +1,64 @@
+# Inventory Object Proposal
+
+## Motivation
+The proposal is to seperate the Application CRD's object selector and group kinds into a separate object. Separating the resource grouping and Application info allows us to reuse resoruce grouping (inventory object) in other controllers and tools. The Inventory object is a descriptive object and needs no controller and status field.
+
+## Summary
+
+- Introduce an Inventory object and a data format that is used to group objects
+- Allow selector and groupKind based grouping
+- Allow explicit object list based grouping
+- Reference array of Inventory objects in Application object
+
+## Proposal
+
+We propose to introduce a separate Inventory/Grouping object. It could be a new API Type or use ConfigMap to achieve the same. 
```

Comment:
Do you plan to create an 'instance' of the inventory ? If not how configmap could be an alternative ?